packages/@aws-cdk/aws-servicecatalog/lib/common.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * Constant for the context directory to store retained ProductStack templates.
+ */
+export const PRODUCT_STACK_CONTEXT_DIRECTORY = 'product-stack-context';
```

Comment:
```suggestion
export const PRODUCT_STACK_CONTEXT_DIRECTORY = 'product-stack-history';
```

?

Reply:
If we are changing it to `fromProductStackSnapshot`. Shouldn't this be `product-stack-snapshots` as well?